Controller/CRUDController.php

```diff
@@ -221,7 +221,8 @@ public function deleteAction($id)
     /**
      * Edit action.
      *
-     * @param int|string|null $id Optional ID to use instead of globally provided default.
+     * @param int|string|null $id Optional ID representing the main admin subject.
```

Comment:
Thanks for the explanation @rande !

The comment I'm talking about was added very recently by @curry684 , with this PR comment : 

> The CRUDController::editAction accepted an $id parameter which was always overwritten, serving 99% of use cases just fine, but not the exceptional case it was added for.

But no explanation what the exceptional case is :( Can you help @curry684 ? I'll dig a bit more in the git log to try to get the rationale behind this.

@Soullivaneuh : indeed the comment is rightly refactored : the previous comment does not explain the why either.